CHANGES.md

```diff
@@ -1,5 +1,17 @@
 # Changes
 
+## edge-22.9.1
+
+This release adds the `linkerd.io/trust-root-sha256` annotation to all injected
+workloads; allowing predictable comparison of all workloads' trust anchors via
+the Kubernetes API.
```

Comment:
nit: or comma. As written, the second part is not a correct sentence, so the semicolon doesn't belong.

```suggestion
This release adds the `linkerd.io/trust-root-sha256` annotation to all injected
workloads allowing predictable comparison of all workloads' trust anchors via
the Kubernetes API.
```